integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -335,7 +339,8 @@ def test_hash_reduction_decimal_overflow_sum(precision):
         # some optimizations are conspiring against us.
         conf = {'spark.rapids.sql.batchSizeBytes': '128m'})
 
-@pytest.mark.parametrize('data_gen', [_longs_with_nulls], ids=idfn)
+@allow_non_gpu("ShuffleExchangeExec")
```

Comment:
I don't understand why this is here.  Do we not support shuffling a particular datatype yet support aggregations on it?

Reply:
Thanks for reviewing. I have added Arrays to ShuffleExchangeExec.